hack/lib/test_lib.sh

```diff
@@ -3,6 +3,13 @@ function listPkgs() {
 }
 
 function listFiles() {
+	# make it work with composed gopath
+	for gopath in ${GOPATH//:/ }; do
+		if [[ $(pwd) =~ $gopath ]]; then
```

Comment:
Quote both operands and `$gopath` below.

Reply:
updated!